USFMToolsSharp/HtmlRenderer.cs

```diff
@@ -137,6 +137,12 @@ private string RenderMarker(Marker input)
                         output.Append(RenderMarker(marker));
                     }
                     output.AppendLine($"</span>");
+
+                    // New Line after each Verse
+                    if (ConfigurationHTML.separateVerses)
+                    {
+                        output.AppendLine("<br class=\"versebreak\"/>");

```

Comment:
I don't think you need the css class.

Reply:
Yeah you're right. It isn't considered good practice attach styling to the <br> tag.

Reply:
@BryanHo10 So we should probably remove it?

Reply:
Oops, sorry. Forgot to push it.